!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  ambient_noise: 2
  ambient_noise_at_signal_0: 0.05
  ambient_noise_at_signal_1: 0.01
  ambient_noise_energy_loss: 0.05
  ambient_noise_energy_max: 0.08
  ambient_noise_energy_min: 0.05
  ambient_noise_uniform_max: 0.15
  beam_sample_dist: 2
  beam_sample_dist_normal_p_in_cone: 0.99
  beam_width: 2.0
  energy_max: 0.72
  energy_min: 0.0
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      ambient_noise: 2
      ambient_noise_at_signal_0: 0.05
      ambient_noise_at_signal_1: 0.01
      ambient_noise_energy_loss: 0.05
      ambient_noise_energy_max: 0.08
      ambient_noise_energy_min: 0.05
      ambient_noise_uniform_max: 0.15
      beam_sample_dist: 2
      beam_sample_dist_normal_p_in_cone: 0.99
      beam_width: 2.0
      energy_max: 0.72
      energy_min: 0.0
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      multipath_threshold: 0.5
      n_cells: 3424
      n_reflections: 4
      n_samples: 10
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      particle_noise: false
      particle_noise_exp_mu: 15.0
      range_max: 600.0
      range_min: 0.0
      record_multi_path: false
      record_multi_reflection: true
      resolution: 0.0595238
      scroll_image: 0
      signal_denoising: 1
      signal_denoising_gaussian_mode: 0.5
      signal_denoising_gaussian_width: 50
      signal_denoising_mb_mode: 0.4
      signal_denoising_mb_width: 50
      signal_denoising_triangular_mode: 0.1
      signal_denoising_triangular_width: 23
      state: true
      type: ''
      z_offset: 0.0
    state: []
  multipath_threshold: 0.5
  n_cells: 3424
  n_reflections: 4
  n_samples: 10
  particle_noise: false
  particle_noise_exp_mu: 15.0
  range_max: 600.0
  range_min: 0.0
  record_multi_path: false
  record_multi_reflection: true
  resolution: 0.0595238
  scroll_image: 0
  signal_denoising: 1
  signal_denoising_gaussian_mode: 0.5
  signal_denoising_gaussian_width: 50
  signal_denoising_mb_mode: 0.4
  signal_denoising_mb_width: 50
  signal_denoising_triangular_mode: 0.1
  signal_denoising_triangular_width: 23
  z_offset: 0.0
state: []
